src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -580,26 +580,51 @@ public Locomotive repetitiveCenterSwipe(SwipeElementDirection direction, int swi
     //endregion Generic swipes
 
     //region Directional swipes
+
+    @Deprecated
```

Comment:
Nice to have: information about the new methods, i.e. "use `scrollDown() to achieve this behavior`

Reply:
Fixed in fe7bf09709c1f8e139ed6bf324b6802244aecc35